requirements.txt

```diff
@@ -1,33 +1,11 @@
-addict==2.2.1
-cachetools==4.1.1
-cython==0.29.21
-decorator==4.4.2
-future==0.18.2
-future-fstrings==1.2.0
-h5py==2.10.0
-joblib
-Keras-Preprocessing==1.1.2
+addict
+future-fstrings
```

Comment:
Can we remove future-fstrings? I think we dropped support for python 3.5. Probably requires changing the encoding of some files. What do you think?

Reply:
Yes, sounds good. Its unnecessary for python > 3.5

Reply:
Done